content/docs/dataform-web/how-to-guides/environments.md

```diff
@@ -66,47 +88,88 @@ Once your team or project outgrows the default environment configuration, a typi
-        "defaultSchema": "dataform_prod",
-        "assertionSchema": "dataform_assertions_prod"
+        "defaultDatabase": "dataform_production"
       }
     }
   ]
 }
 ```
 
-This configuration defines two environments. The `staging` environment runs the project's schedules at the latest version of the
-project's code (as exists on the `master` branch). When you have confirmed that the code works as expected, you can push those
-schedules to production by merging the `master` branch into the `production` branch.
+To update the version of the project running in `production`, change the value of `commitSha`, and then push that change to your `master` branch. On GitHub the commit sha can be found by opening the project page, clicking commits, then copying the desired sha from the presented list of commits.
 
-### Enforcing tighter control on changes to an environment
+### Configuration via branches
 
-In some cases it may be preferred to tightly control changes to an environment. For example, you might want to enforce
-that code is only pushed to a `production` environment after a pull request and/or code review.
+<div className="bp3-callout bp3-icon-info-sign" markdown="1">
+  Branches seen on dataform are not remote git branches, but are only used for local development. Because of this, a <code>branch</code> specified in a <code>gitReference</code> won't point a branch on dataform, with the exception of the master branch.
+</div>
+
+In this configuration:
+
+- the `production` environment is locked to a specific remote git branch
+
+- the `staging` environment runs the project's schedules at the latest version of the project's code (as exists on the `master` branch)
 
 ```json
 {
   "environments": [
+    {
+      "name": "staging",
+      "gitReference": {
+        "branch": "master"
+      },
+      "configOverride": {
+        "defaultDatabase": "dataform_staging"
+      }
+    },
     {
       "name": "production",
       "gitReference": {
-        "commitSha": "67bed6bd4205ce97fa0284086ed70e5bc7f6dd75"
+        "branch": "production"
+      },
+      "configOverride": {
+        "defaultDatabase": "dataform_production"
       }
     }
   ]
 }
 ```
 
-This example locks the `production` environment to a specific Git commit. To update the version of the project running in
-`production`, you must change the value of `commitSha`, and then push that change to your `master` branch (which may
-require a pull request, depending on your project's settings). This process has the nice property of requiring that any
-change to `production` is recorded in an audit trail (i.e. your project's Git commits).
+To update the version of the project running in `production`, merge the `master` branch in to the `production` branch.
+
+### Config schema overriding
+
+Redshift and Snowflake do not configure multiple databases like alternative providers. Overriding the schema suffix instead of databases are a good solution to this. For example:
```

Comment:
Snowflake does. I wouldn't mention the warehouse specifically, I'd just say:
"Another approach to separating `production` and `staging` data is to append a suffix to schemas in the staging environment. For example:"

Reply:
👍